src/pages/kb/user-guide/querying/query-parameters.md

```diff
@@ -120,3 +118,33 @@ WHERE org_id = {{org_id}} AND created_at > '{{start_date}}'
 ```
 
 We use two parameters: `{{org_id}}` and `{{start_date}}`.
+
+## Parameter Mapping on Dashboards
```

Comment:
Worth adding a callout box here saying that this will be part of the next Redash release (7.0) and already available in the hosted platform.